pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -1361,7 +1361,7 @@ const (
 	DefTiDBOptRangeMaxSize                       = 64 * int64(size.MB) // 64 MB
 	DefTiDBCostModelVer                          = 2
 	DefTiDBServerMemoryLimitSessMinSize          = 128 << 20
-	DefTiDBMergePartitionStatsConcurrency        = 1
+	DefTiDBMergePartitionStatsConcurrency        = 2
```

Comment:
Add a todo comment to change it back 1 (This is a temp value just for test concurrency partition stats)

Reply:
I have been reverted.